src/components/books.css

```diff
@@ -1,14 +1,110 @@
-.book {
```

Comment:
- [ ] You are almost there! However, your styling is looking fairly different from the template provided, I would suggest using the template fonts, colors, and positioning, the spacing between the cards is missing and form still needs some details:

|Your Design|Template|
|---|---|
|<img width="1425" alt="Screen Shot 2022-07-16 at 9 29 31 PM" src="https://user-images.githubusercontent.com/83717285/179381422-bdf3f631-962a-4fcd-a6a5-dd7568ad3ac2.png">|<img width="1038" alt="Screen Shot 2022-07-16 at 9 28 01 PM" src="https://user-images.githubusercontent.com/83717285/179381428-c7f11441-391e-43af-a57d-583b1d4ec211.png">|